### ISSUE TITLE
Implement tanh similar to sinh and cosh

### ISSUE THREAD

**Issue by ingomueller-net** [RumbleDB/rumble#707]
~~Expose sinh and cosh the same way as sin and cos are exposed.~~
Implement tanh similar to sinh and cosh

**Comment by ghislainfourny** [comment]
Good point, and tanh. These are not standard, but since we have a Rumble namespace that can have more builtin functions, this should be no problem.

**Comment by mschoeb** [comment]
`sinh` and `cosh` have been implemented in the meantime. `tanh` not yet, but I think this could be a good first issue to get started for future students so I will leave it open for now
